src/tests/components/DeleteModal.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { DeleteModal } from '../../components/Articles/DeleteModal';
+
+describe('DeleteModal Component', () => {
+  let wrapper;
+  const confirmDelete = true;
+  const deleteArticle = jest.fn();
+  const slug = 'react-redux';
+
+  const getEvent = (name = '', value = '') => ({
+    preventDefault: jest.fn(),
+    target: {
+      name,
+      value,
+    },
+  });
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <DeleteModal
+        slug={slug}
+        confirmDelete={confirmDelete}
+        deleteArticle={deleteArticle}
+      />,
+    );
+  });
+
+  it('should render the Articles Component correctly', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should call deleteArticle when handleClick is called', () => {
+    wrapper.instance().handleClick((getEvent()));
+    expect(deleteArticle).toBeCalled();
+  });
+
+  it('should set redirect to true when user has confirmed delete', () => {
```

Comment:
`Set state redirect value to true on successfully deleting an article`.

Reply:
I have made the changes as you suggested